src/front-end/src/components/ForgotPassword.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+import { Form, Button, Input } from 'antd';
+import { resetPassword } from '../helper-methods';
+
+class ForgotPassword extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+    };
+    this.localResetPassword = this.localResetPassword.bind(this);
+    this.handleEmailChange = this.handleEmailChange.bind(this);
+  }
+
+  localResetPassword() {
+    // TODO: implement some validation here
+    resetPassword(this.state.email);
+  }

```

Comment:
I didn't realize that the 'Confirm' button was working, and so I clicked 'confirm' like 5 times, checked my email and saw that it sent 5 password reset emails (oops haha). Can a pop-up be added here to prevent that? Something along the lines of : window.alert('Please check your email to reset your password!');

Reply:
I agree Katrina, but I think in the message we could be a bit clearer about the fact that we already sent the email. Maybe something like "We sent you an email with a link to reset your password."?

Reply:
done